src/authV2/azext_authV2/_help.py

```diff
@@ -0,0 +1,113 @@
+  - name: Configure the app to redirect unauthenticated requests to the Facebook provider
+    text: >
+        az webapp auth update -g myResourceGroup -n myUniqueApp --redirect-provider Facebook
+  - name: Configure the app to listen to the forward headers X-FORWARDED-HOST and X-FORWARDED-PROTO
+    text: >
+        az webapp auth update -g myResourceGroup -n myUniqueApp --proxy-convention Standard
+"""
+
+helps['webapp auth set'] = """
+type: command
+short-summary: Sets the authentication settings for the webapp in the v2 format, overwriting any existing settings.
+examples:
+  - name: Set the json saved in file auth.json as the auth settings for the web app, overwriting any existing settings.
+    text: >
+        az webapp auth set -g myResourceGroup -n myUniqueApp --body @auth.json
+"""
+
+helps['webapp auth config-version show'] = """
+type: command
+short-summary: Show the configuration version of the authentication settings for the webapp.
+examples:
+  - name: Show the configuration version of the authentication settings for the webapp (autogenerated)
+    text: >
+        az webapp auth config-version show --name MyWebApp --resource-group MyResourceGroup
+"""
+
+helps['webapp auth config-version revert'] = """
+type: command
+short-summary: Reverts the configuration version of the authentication settings for the webapp from v2 to v1 (classic).
+examples:
+  - name: Revert the configuration version of the authentication settings for the webapp from v2 to v1 (classic) (autogenerated)
+    text: >
+        az webapp auth config-version revert --name MyWebApp --resource-group MyResourceGroup
+"""
+
+helps['webapp auth config-version upgrade'] = """
+type: command
+short-summary: Upgrades the configuration version of the authentication settings for the webapp from v1 (classic) to v2.
+examples:
+  - name: Upgrades the configuration version of the authentication settings for the webapp from v1 (classic) to v2 (autogenerated)
+    text: >
+        az webapp auth config-version upgrade --name MyWebApp --resource-group MyResourceGroup
+"""
+
+helps['webapp auth-classic'] = """
+type: group
+short-summary: Manage webapp authentication and authorization in the classic format.
+"""
+
+helps['webapp auth-classic show'] = """
+type: command
+short-summary: Show the authentification settings for the webapp in the classic format.
+examples:
+  - name: Show the authentification settings for the webapp. (autogenerated)
+    text: az webapp auth show --name MyWebApp --resource-group MyResourceGroup
+    crafted: true
+"""
+
+helps['webapp auth-classic update'] = """
+type: command
+short-summary: Update the authentication settings for the webapp in the classic format.
+examples:
+  - name: Enable AAD by enabling authentication and setting AAD-associated parameters. Default provider is set to AAD. Must have created a AAD service principal beforehand.
```

Comment:
From a branding perspective, I think we're supposed to avoid "AAD" in descriptive text and instead use "Azure AD" or "Azure Active Directory". Matthew might know what the official guidance is on this.